pages/linux/pvs.md

```diff
@@ -0,0 +1,31 @@
+# pvs
+
+> Display information about physical volumes.
```

Comment:
Is there any links we can put here?

Reply:
man-pages are the only comprehensive documentation i could find for `pvs`. All the online resources are essentially copying man-pages.

Reply:
Thank you for replying, leave it blank is ok.